drivers/sensor/ams/tsl2540/tsl2540.c

```diff
@@ -17,10 +17,10 @@
 #include <zephyr/sys/byteorder.h>
 #include <zephyr/sys/util.h>
 
-#define TSL2540_INTEGRATION_TIME_MS	(2.81)
-#define TSL2540_DEVICE_FACTOR		(53.0)
+#define TSL2540_INTEGRATION_TIME_MS (2.81)
+#define TSL2540_DEVICE_FACTOR       (53.0)
```

Comment:
There are a lot of unrelated whitespace changes like this that add noise to the commit. Please revert them to reduce the diff to only the relevant gain/resolution changes.